### PR TITLE
request create from byte[] range

### PR DESCRIPTION
**Pull request by longbai**

### CODE REVIEW

**Comment by longbai** [comment]
have signed CLA.

File: okhttp/src/main/java/com/squareup/okhttp/RequestBody.java [hunk 1]
```diff
@@ -74,6 +74,26 @@ public static RequestBody create(final MediaType contentType, final byte[] conte
     };
   }
 
+  /** Returns a new request body that transmits {@code content}. */
```
Comment (swankjesse) [inline okhttp/src/main/java/com/squareup/okhttp/RequestBody.java]: Can you fix the overload on line 60 to call through to this? No sense in having more code than we need.

**Comment by swankjesse** [comment]
Works for me. Just one nitpick.

**Comment by longbai** [comment]
done.

**Comment by swankjesse** [comment]
Awesome. (Sorry for sitting on this!)

CLA looks good. My last request: do you mind squashing this into a single commit? See http://rebaseandsqua.sh for instructions.

**Comment by longbai** [comment]
done

**Comment by swankjesse** [comment]
Thanks!

**Pull request merged**